WorksheetB

```diff
@@ -0,0 +1,31 @@
+Word Game Psuedocode
```

Comment:
Flowchart looks OK, but some lines are missing arrows which makes the flow direction difficult to follow.